src/actors/client/ActorProxy.ts

```diff
@@ -1,38 +0,0 @@
-import { AbstractActor } from "../..";
-import DaprClient from "../../implementation/Client/DaprClient";
-import Class from "../../types/Class";
-import ActorId from "../ActorId";
-import ActorProxyFactory from "./ActorProxyFactory";
-
-export default class ActorProxy<T extends AbstractActor> {
```

Comment:
Why are you removing ActorProxy? Also, this would be a breaking change, not?

Reply:
There is a ProxyBuilder now that implements Proxy itself. This was old deprecated code and was not functional